docs/en/concepts-and-designs/event.md

```diff
@@ -55,10 +55,49 @@ The end time of the event. This field may be empty if the event has not ended ye
 **NOTE:** When reporting an event, you typically call the report function twice, the first time for starting of the event and the second time for ending of the event, both with the same UUID.
 There are also cases where you would already have both the start time and end time. For example, when exporting events from a third-party system, the start time and end time are already known so you may simply call the report function once.
 
+## How to Configure Alarms for Events
+
+Events are derived from metrics, and can be the source to trigger alarms. For example, if a specific event occurs for a
+certain times in a period, alarms can be triggered and sent.
+
+Every event has a default `count = 1`, when `n` events with the same name are reported, they are aggregated
+into `count = n` as follows.
+
+```
+Event{name=Unhealthy, source={service=A,instance=a}, ...}
+Event{name=Unhealthy, source={service=A,instance=a}, ...}
+Event{name=Unhealthy, source={service=A,instance=a}, ...}
+Event{name=Unhealthy, source={service=A,instance=a}, ...}
+Event{name=Unhealthy, source={service=A,instance=a}, ...}
+Event{name=Unhealthy, source={service=A,instance=a}, ...}
+```
+
+will be aggregated into
+
+```
+Event{name=Unhealthy, source={service=A,instance=a}, ...} <count = 6>
+```
+
+so you can configure the following alarm rule to trigger alarm when `Unhealthy` event occurs more than 5 times within 10
+minutes.
+
+```yaml
+rules:
+  unhealthy_event_rule:
+    metrics-name: Unhealthy
+    threshold: 5
```

Comment:
What threshold means? how many events per minute?

Reply:
> What threshold means? how many events per minute?

yes

Reply:
Then, is 5 a good number of examples here? I think usually event threshold should be 1?

Reply:
> Then, is 5 a good number of examples here? I think usually event threshold should be 1?

Healthiness check is usually a scheduled task, they may be unhealthy for the first few times, and can be unhealthy occasionally due to network jitter, I feel 5 is acceptable as just an example

Reply:
OK, then at least, this would be taken as a widely adopted example. So, put some explanation on the document.